Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -349,7 +360,9 @@ def retrieve_versions_by_arch(formula_or_cask:, repositories:, name:)
               current_version_value = Version.new(loaded_formula_or_cask.version)
             end
 
-            livecheck_latest = livecheck_result(loaded_formula_or_cask)
+            livecheck_latest, livecheck_latest_throttled = livecheck_result(loaded_formula_or_cask)
+            # TODO: Pass down `livecheck_latest` info to print output for throttled formulae or casks
+            livecheck_latest = livecheck_latest_throttled if livecheck_latest_throttled
```

Comment:
Given that `brew bump` doesn't really need unthrottled info (and it is available via `livecheck`), I'll go ahead an simplify this (and handle in function)
```suggestion
            livecheck_latest = livecheck_result(loaded_formula_or_cask)
```